crates/cli/src/subcommands/call.rs

```diff
@@ -37,50 +38,36 @@ pub fn cli() -> clap::Command {
         .after_help("Run `spacetime help call` for more detailed information.\n")
 }
 
-pub async fn exec(mut config: Config, args: &ArgMatches) -> Result<(), Error> {
+pub async fn exec(config: Config, args: &ArgMatches) -> Result<(), Error> {
     eprintln!("{}\n", UNSTABLE_WARNING);
-    let database = args.get_one::<String>("database").unwrap();
     let reducer_name = args.get_one::<String>("reducer_name").unwrap();
     let arguments = args.get_many::<String>("arguments");
-    let server = args.get_one::<String>("server").map(|s| s.as_ref());
-    let force = args.get_flag("force");
-
-    let anon_identity = args.get_flag("anon_identity");
-
-    let database_identity = database_identity(&config, database, server).await?;
-
-    let builder = reqwest::Client::new().post(format!(
-        "{}/database/call/{}/{}",
-        config.get_host_url(server)?,
-        database_identity.clone(),
-        reducer_name
-    ));
-    let auth_header = get_auth_header(&mut config, anon_identity, server, !force).await?;
-    let builder = add_auth_header_opt(builder, &auth_header);
-    let describe_reducer = util::describe_reducer(
-        &mut config,
-        database_identity,
-        server.map(|x| x.to_string()),
-        reducer_name.clone(),
-        anon_identity,
-        !force,
-    )
-    .await?;
+
+    let conn = parse_req(config, args).await?;
+    let api = ClientApi::new(conn);
+
+    let database_identity = api.con.database_identity;
+    let database = &api.con.database;
```

Comment:
You get it from the `api.con` but that does not take into account the args. What's going on here?

Reply:
Looks like it does take into account the args via the `parse_req` function. That is just confusingly in the `sql` `mod`.

Reply:
Yeah, I dunno why it's there. I have a follow-up branch that moves most of the cli to use ClientApi and renames parse_req that I was gonna try and squeeze into http-glowup, but I decided against it. I'll probably open it post-1.0.

Reply:
Roger. Thank you!